lib/presentation/features/home/home_screen.dart

```diff
@@ -19,20 +18,27 @@ class HomeScreen extends StatelessWidget {
         title: const Text("MarvellApp"),
       ),
       body: BlocProvider(
-        create: (_) =>
-            HomeBloc(getCharactersUseCase: GetCharactersUseCase(repository: DioMarvellRepository()))..add(GetHeroes()),
+        create: (_) => HomeBloc(getCharactersUseCase: GetCharactersUseCase(repository: DioMarvellRepository()))
+          ..add(ReadyForData()),
         child: BlocBuilder<HomeBloc, HomeState>(
           builder: (context, state) {
             Widget child = const Center(
               child: CircularProgressIndicator(),
             );
```

Comment:
Probably you no need this `child` value - you can use your widget in place (line 32)

Reply:
done